dm/syncer/metrics/validator_metrics.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/pingcap/tiflow/dm/pkg/metricsproxy"
+)
+
+var (
+	ValidatorErrorCount = metricsproxy.NewCounterVec(
```

Comment:
error row change maybe re-find on recover from checkpoint. and using counter is hard to add alerts since we may not find error row change uniformly in time.

maybe use a gauge, and update it on validator status routine.